metadata/src/main/java/com/redhat/lightblue/metadata/Indexes.java

```diff
@@ -61,6 +63,20 @@ public boolean isEmpty() {
         return indexes.isEmpty();
     }
 
+    public boolean isCaseInsensitiveKey(Path path) {
+        return getCaseInsensitiveIndexes().stream()
+                .anyMatch(i -> i.getField().equals(path));
+
+    }
+
+    public List<IndexSortKey> getCaseInsensitiveIndexes() {
```

Comment:
This is only used in one place right now, in Translator.  It's used to populate any hidden fields that have a case insensitive index.  We could just move this there if you don't see any need to have this here.